lib/kite_connect_ex/request.ex

```diff
@@ -18,6 +18,16 @@ defmodule KiteConnectEx.Request do
   @spec api_version() :: String.t()
   def api_version, do: @api_version
 
+  @spec get_csv(String.t(), map() | nil, list, list) :: Response.success() | Response.error()
+  def get_csv(path, query, headers, opts \\ []) do
```

Comment:
the request module is just a wrapper around all HTTP calls. Can you create a separate module for handling CSV request/response? How are these requests different from regular HTTP requests?

Reply:
It just doesnt use do a Jason decode on the response ,  I was going to convert the csv dump to json but I did in heimdall code instead , where should I be doing this ?  @nitinstp23

Reply:
@nitinstp23 can I pass an ignore optional flag instead so that it doesnt do json decode ?

Reply:
can you pattern match on the response headers in the `KiteConnectEx.Response.parse_response` function?

```elixir
  def parse_response(%HTTPoison.Response{} = response) do
  def parse_response(%HTTPoison.Response{} = response) do
```